document/js-api/index.bs

```diff
@@ -1145,8 +1151,10 @@ The algorithm <dfn>ToWebAssemblyValue</dfn>(|v|, |type|) coerces a JavaScript va
         1. Return [=ref.extern=] |r|.
     1. If |v| is null,
         1. Let |r| be [=ref.null=] |heaptype|.
-    1. Else if |v| [=is a Number=] and |v| is equal to ? [$ToInt32$](|v|) and [=ℝ=](|v|) < 2<sup>30</sup> and [=ℝ=](|v|) ⩾ -2<sup>30</sup>,
-        1. Let |r| be [=ref.i31=] |v|.
+    1. Else if |v| [=is a Number=] and |v| is equal to [=?=] [$ToInt32$](|v|) and [=ℝ=](|v|) < 2<sup>30</sup> and [=ℝ=](|v|) ⩾ -2<sup>30</sup>,
+        1. Let |i31| [=?=] [$ToInt32$](|v|).
+        1. Let |u31| be the unsigned integer such that |i31| is [=signed_32=](|i|).
```

Comment:
Did you mean, `[=signed_32=](|i31|).` here?

Reply:
Hmm actually, is `signed_32` right here?

For example, with `i31 = -1` we must therefore have `u31 = (2^32) - 1` which is larger than the highest u31 value.

Reply:
Yes, you are right, has to be signed_31. Fixed.